src/components/WishlistPage/WishlistPageContainer.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+
+import * as actions from '../../actions/actionsWishlist';
+import WishlistPage from './WishlistPage';
+
+function mapStateToProps(state) {
+  return { wishlist: state.reducerWishlist.wishlist };
+}
+
+export default connect(mapStateToProps, actions)(WishlistPage);
```

Comment:
Where is **mapDispatchToProps**? Why actions instead?

Reply:
Isn't it just another way of doing the same?

Reply:
You need to use bindActionCreators inside mapDispatchToProps so that they would be bind to dispatch function and could make some effect on store or use dispatch inside mapDispatchToProps to and dispatch actions using it. My explanation can be messy but here is documentation and I believe it will clarify everything for you https://react-redux.js.org/api/connect

Reply:
The bindActionCreator was so popular, that react-redux guys added autobinding to dispatch, if you're passing an object as second arg to mdtp, BUT there is still sense to use bindActionCreators for grouping store actions in one place (under storeActions key)
e.g. 
``` 
storeActions: bindActionCreator({actions}, dispatch)
```